web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -60,11 +62,237 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
     $scope.loadPage(false);
   };
 
+  $scope.selectedItems = [];
+
+  $scope.showActionBar = false;
+
+  $scope.$watch('selectedItems', function (newValue) {
+    $scope.badgeSelection = {};
+    $scope.awardBadgeButtonEnabled = false;
+  });
+
+  $scope.$watch('selectedItems.length', function (newValue) {
+    $scope.badgeSelection = {};
+    $scope.awardBadgeButtonEnabled = false;
+    $scope.showActionBar = newValue > 0;
+  });
+
+  $scope.adultRoles = [
```

Comment:
THis is so similar to what's sued in cd_choose-role
A todo : refactor may be good ?